comp/languagedetection/bundle.go

```diff
@@ -1,26 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I'm not sure about the purpose of this file.
MockBundle() is not used anywhere, and Bundle() is only used in the test file below.
It's the only test that failed after the refactor.